.github/workflows/python.yml

```diff
@@ -112,6 +112,43 @@ jobs:
         continue-on-error: true
         run: archery docker push ${{ matrix.image }}
 
+  build-example:
+    name: ${{ matrix.title }}
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        name:
+          - minimal-fedora-conda
+          - minimal-fedora-venv
+          - minimal-ubuntu-conda
+          - minimal-ubuntu-venv
+        include:
+          - name: minimal-fedora-conda
+            image: minimal-fedora-conda
+            title: Fedora Conda Minimal Build
```

Comment:
Could you please use `<arch> <os> <version> <build-name>` title to better align with the rest of the build titles?